src/net.h

```diff
@@ -755,6 +778,10 @@ class CNode
     // Our address, as reported by the peer
     CService addrLocal;
     mutable CCriticalSection cs_addrLocal;
+
+    /** Our current dandelion destination */
+    CConnman::DandelionPeer* m_dandelion_destination /* GUARDED_BY(CConnman::cs_dandelion_peers) */ {nullptr};
```

Comment:
Any drawbacks from uncommenting the `GUARDED_BY(…)` annotation?

Reply:
The drawback was that it didn't compile for me